src/ui/layout/qgslayoutscalebarwidgetbase.ui

```diff
@@ -557,10 +600,15 @@
    <header>qgsscrollarea.h</header>
    <container>1</container>
   </customwidget>
+  <customwidget>
+   <class>QgsSpinBox</class>
+   <extends>QSpinBox</extends>
+   <header>qgsspinbox.h</header>
+  </customwidget>
   <customwidget>
    <class>QgsCollapsibleGroupBoxBasic</class>
    <extends>QGroupBox</extends>
-   <header>qgscollapsiblegroupbox.h</header>
+   <header location="global">qgscollapsiblegroupbox.h</header>
```

Comment:
location="global" -> that's a QtCreator change?

Reply:
Yep -- seems to randomly turn this on or off based on ... ?